src/Knp/Component/Pager/PaginatorInterface.php

```diff
@@ -21,6 +21,9 @@ interface PaginatorInterface
     public const FILTER_VALUE_PARAMETER_NAME = 'filterValueParameterName';
     public const FILTER_FIELD_WHITELIST = 'filterFieldWhitelist';
     public const DISTINCT = 'distinct';
+    public const FORCE_VALID_PAGINATION_PARAMETERS = 'forceValidPaginationParameters';
+    public const DEFAULT_LIMIT = 10;
```

Comment:
I would add a line break here to emphasize the difference in nature of these constants